work.py

```diff
@@ -397,9 +403,10 @@ def sync_timesheetline(cls, activities):
                             work=activity.resource.rec_name))
                     continue
 
-                timesheet_line = TimesheetLine()
-                timesheet_line.activity = activity
-                timesheet_line.work = activity.resource.timesheet_works[0]
+                if activity.resource.timesheet_works:
```

Comment:
This `if` is unnecessary because the previous ìf` already handled the case of  `timesheet_works` being empty (and `continue` in that case).